core/docz-components/src/components/Playground/index.tsx

```diff
@@ -90,27 +139,21 @@ export const Playground = ({
         theme={theme}
       >
         <div sx={styles.previewWrapper}>
-          <LivePreviewWrapper showingCode={showingCode}>
-            {showLivePreview && (
-              <LivePreview sx={styles.preview} data-testid="live-preview" />
-            )}
-          </LivePreviewWrapper>
-          <div sx={styles.buttons}>
-            <button sx={styles.button} onClick={() => copy(code)}>
-              <Icons.Clipboard size={12} />
-            </button>
-            <button sx={styles.button} onClick={toggleCode}>
-              <Icons.Code size={12} />
-            </button>
-          </div>
+          {preview}
+          {buttons}
         </div>
-        {showLiveError && (
-          <LiveError sx={styles.error} data-testid="live-error" />
-        )}
-        {showingCode && (
-          <div sx={styles.editor(theme)}>
-            <LiveEditor data-testid="live-editor" />
-          </div>
+        {error}
+        {editor}
+        {showFullscreen && (
+          <Dialog
+            title="Live Preview"
+            onClose={toggleFullscreen}
+            data-testid="livepreview-fullscreen"
+            width={dialogWidth}
+          >
+            <DialogActions onChangeSize={onChangeSize} />
+            {preview}
```

Comment:
Shouldn't we also have `error` and `editor` here ?

Reply:
Sure, can add it